test/statements/match4.py

```diff
@@ -0,0 +1,59 @@
+match point:
```

Comment:
Can you add a test with a more complicated expression in `match`?

Reply:
Also things like

```
case ["drop", *objects]:
```

I suggest to go though the examples here https://peps.python.org/pep-0636/

Reply:
I've added a more complex expression for match and some more variety for cases.

Generally the expressions there are not special for this grammar, so I don't expect them to mess things up. The only thing that matters is that the expressions start with something valid and not, say, `^` or `%`.